.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "cmake.sourceDirectory": "C:/Users/krisk/Desktop/Dev/Flutter-Training-PI/windows/flutter"
+    "cmake.sourceDirectory": "C:/Users/krisk/Desktop/Dev/Flutter-Training-PI/windows/flutter",
```

Comment:
this is a file that probably shouldn't be checked in because it has an actual path in it